cpu/stm32/stm32_mem_lengths.mk

```diff
@@ -271,6 +281,8 @@ else ifeq ($(STM32_TYPE), U)
     ifneq (, $(filter $(STM32_MODEL2), 7 8))
       RAM_LEN = 768K
       SRAM4_LEN = 16K
+      BACKUP_RAM_ADDR = 0x40036400
```

Comment:
Is it correct to use the "Non-secure boundary address" here?

Reply:
@aabadie could give it a try

Reply:
I think the `stm32u5` lacks `periph_rtc` support yet. I got compiling errors when I tried to compile the backup RAM test for the board with that CPU. That was the reason why I added `FEATURES_REQUIRED += periph_rtc` to the test.
At least the hello world example with a variable in backup RAM did compile.